app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayout.kt

```diff
@@ -238,24 +246,30 @@ class OmnibarLayout @JvmOverloads constructor(
         )[OmnibarLayoutViewModel::class.java]
     }
 
+    // One OmnibarLayout is always removed from the view hierarchy, we should cancel any flow subscriptions
+    fun onRemoved() {
```

Comment:
Either bottom or the top omnibar is always removed but its ViewModel flow subscriptions would keep collecting & rendering state for no reason.